### DISCUSSION TITLE
Use case: React project

### DISCUSSION THREAD

**Discussion by tdnelson2** [privateOmega/html-to-docx#147] [General]
> Currently it doesn't work with browser directly, but it was tested against React.

Can you explain what this means. Were you able to get this working in a React project? If so, are you willing to provide an example?

**Comment by privateOmega** [comment]
@tdnelson2 Sorry for the response after a year, I never saw this discussion, but theres an [example already in the repo](https://github.com/privateOmega/html-to-docx/tree/master/example/react-example)
